client/signup/is-flow.ts

```diff
@@ -1,7 +1,4 @@
-export const isP2Flow = ( flowName: string ) => {
-	return flowName === 'p2';
```

Comment:
There are similar checks in the codebase that don't use this helper, let's clean them up. One example:

https://github.com/Automattic/wp-calypso/blob/42613bf497329dfc20989fa962ec2cc65e5dfda3/client/lib/signup/step-actions/index.js#L1178

Reply:
The provided instance has been removed by 58d8d1b6509. I don't see other instances from a grep. If I'm missing anything, please surely let me know :)